test/utils/helpers.ts

```diff
@@ -1,26 +1,92 @@
 import { BigNumber } from 'ethers';
-import { ethers } from 'hardhat'  
+import { ethers } from 'hardhat';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
 
-
-export const zero = ethers.constants.Zero; 
+export const zero = ethers.constants.Zero;
 
 export interface Reward {
   _address: string;
   _amount: BigNumber;
+  _signer?: SignerWithAddress;
 }
 
 /**
- * @notice Sums rewards amounts that are due to voters 
- * @param rewards Array of structs of voters' addresses and amounts 
- * to be transfered 
+ * @notice Sums rewards amounts that are due to voters
+ * @param rewards Array of structs of voters' addresses and amounts
+ * to be transfered
  */
 
```

Comment:
I'm not entirely confident that jsdoc will work when there is a line between the comments and the function. It may be fine but let's follow convention and take this empty line out. 
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html

Reply:
Tested and it still works with this empty line.